kfs-purap/src/main/resources/edu/arizona/kfs/workflow/PurchaseOrderAmendmentDocument.xml

```diff
@@ -0,0 +1,98 @@
+          <status>Closed</status>
+          <status>Disapproved Budget</status>
+          <status>Disapproved C and G</status>
+          <status>Disapproved Change</status>
+          <status>Disapproved Commodity Code</status>
+          <status>Disapproved Purchasing</status>
+          <status>Disapproved Tax</status>
+          <status>Open</status>
+          <status>Pending Amendment</status>
+          <status>Pending Close</status>
+          <status>Pending Payment Hold</status>
+          <status>Pending Remove Hold</status>
+          <status>Pending Reopen</status>
+          <status>Pending Retransmit</status>
+          <status>Pending Void</status>
+          <status>Retired Version</status>
+          <status>Void</status>
+        </category>
+        <category name="INCOMPLETE STATUSES">
+          <status>In Process</status>
+          <status>Awaiting Fiscal Officer Review</status>
+          <status>Awaiting New Unordered Item Review</status>
+          <status>Disapproved New Unordered Item Review</status>
+          <status>Awaiting Purchasing Approval</status>
+          <status>Awaiting Commodity Code Approval</status>
+          <status>Awaiting C and G Approval</status>
+          <status>Awaiting Budget Approval</status>
+          <status>Awaiting Tax Approval</status>
+          <status>Error occurred sending cxml</status>
+          <status>Error occurred sending fax</status>
+          <status>Out for Quote</status>
+          <status>Payment Hold</status>
+          <status>Pending Fax</status>
+          <status>Pending Print</status>
+          <status>Pending cxml</status>
+          <status>Waiting for Department</status>
+          <status>Waiting for Vendor</status>
+        </category>
+      </validApplicationStatuses>
+      <routePaths>
+        <routePath>
+          <start name="AdHoc" nextNode="HasNewUnorderedItems" />
+          <split name="HasNewUnorderedItems" nextNode="ObjectSubTypeCode">
+            <branch name="True">
+              <role name="NewUnorderedItems" nextNode="Join" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOp" nextNode="Join" />
+            </branch>
+            <join name="Join" />
+          </split>
+          <role name="ObjectSubTypeCode" />
+        </routePath>
+      </routePaths>
+      <routeNodes>
+        <start name="AdHoc" />
+        <split name="HasNewUnorderedItems">
+          <type>org.kuali.kfs.sys.document.workflow.SimpleBooleanSplitNode</type>
+        </split>
+        <role name="NewUnorderedItems">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="ObjectSubTypeCode">
+          <qualifierResolverClass>org.kuali.rice.kns.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
```

Comment:
should be:
org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver